server/graphql/types.js

```diff
@@ -70,10 +72,14 @@ const getCustomTypes = (strapi, nexus) => {
 							},
 						};
 
-						// only return published entries
+						// return entries based on publicationState arg or default to only returning published entries when draftAndPublish mode is enabled
```

Comment:
I would prefer to keep the comments in sync between REST and graphql
```suggestion
						// only return published entries by default if content type has draftAndPublish enabled
```